ansible/roles/stack-sunbird/defaults/main.yml

```diff
@@ -204,6 +204,13 @@ taxonomy_service_limit_memory: 512M
 taxonomy_service_reservation_cpu: 0
 taxonomy_service_limit_cpu: 1
 
+# Dial Service Config
```

Comment:
Please remove this section. This is not used anywhere in the helm chart,

Reply:
Also please remove these config sections in the same file.
`# Taxonomy Service Config `
`# Search Service`
`# Assessment Service Config`
`# Content Service Config`

Reply:
done